.github/workflows/flower.yml

```diff
@@ -35,3 +35,9 @@ jobs:
         run: ./dev/check-protos.sh
       - name: Lint + Test (isort/black/docformatter/mypy/pylint/flake8/pytest)
         run: ./dev/test.sh
+      - name: Check if pyproject.toml and requirements.txt are in sync
+        run: |
+          python -m pip install poetry-add-requirements.txt
+          ./dev/check-requirements-txt.sh
+        if: ${{ matrix.python == '3.10.9' }}
```

Comment:
We never do this, so I'd prefer not to start now. Just let it run in each version in the matrix.

Reply:
The tool requires python version >=3.10

Reply:
Got it! Thank you.